rmgpy/data/kinetics/family.py

```diff
@@ -2029,7 +2029,7 @@ def getLabeledReactantsAndProducts(self, reactants, products):
         If RMG cannot find appropriate labels, (None, None) will be returned.
         """
```

Comment:
the docstring should be updated since now the input reactants and products are `Species` type?